web/frontend/src/schema/configurationValidation.ts

```diff
@@ -1,11 +1,15 @@
 import * as yup from 'yup';
 
 const idSchema = yup.string().min(1).required();
-const titleSchema = yup.string().required();
-const formTitleSchema = yup.object({
-  en: yup.string().required(),
-  fr: yup.string(),
-  de: yup.string(),
+const titleSchema = yup.object({
+  En: yup.string().required(),
```

Comment:
Where is this validation enforced?

Reply:
when the schema gets executed upon submission

in pages/form/components/AddQuestionModal.tsx for the subjects and in pages/form/components/UploadFile.tsx for the JSON files and pages/form/components/FormForm.tsx for the form